intTests/test_type_errors/err005.saw

```diff
@@ -0,0 +1 @@
+typedef x = { foo: Int, bar: x };
```

Comment:
Since this commit fixes #2077, it would be worth advertising this in the changelog.

Reply:
Good point. Added in e3bdc488a0d54de6e5c6604605dfa87cacde23e5